src/utils/TaggedCollections.fs

```diff
@@ -711,7 +711,7 @@ namespace Internal.Utilities.Collections.Tagged
 #if ONE 
         | MapOne of 'Key * 'T
 #endif
-        // Note: performance rumour has it that the data held in this node should be
+        // NOTE: performance rumour has it that the data held in this node should be
```

Comment:
Please make an issue to review this and then label it ISSUE  nnn where nnn is the github issue ID.  Then someone can take a look and either remove the comment or make a change.